tests/scenarios/traffic_split_with_apex_service_test.go

```diff
@@ -151,11 +144,11 @@ var _ = Describe(``+
 
 				expectedWeightedCluster := &xds_route.WeightedCluster{
 					Clusters: []*xds_route.WeightedCluster_ClusterWeight{
-						weightedCluster("bookstore-apex", 100),
+						// weightedCluster("bookstore-apex", 100),
```

Comment:
Any reason why we can't just remove this?

Reply:
I felt compelled to leave it here to explicitly show that the apex cluster has been removed -- a documentation of a kind.